solr/CHANGES.txt

```diff
@@ -36,7 +36,8 @@ Improvements
 
 Optimizations
 ---------------------
-(No changes)
+* SOLR-17568: The CLI bin/solr export tool now contacts the appropriate nodes directly for data instead of proxying through one.
```

Comment:
@epugh you might be interested in this aspect as it touches the CLI.  Or perhaps not since you are working on CLI infrastructure and not the business of what they do.

Reply:
Shrug; I found the issue above by seeing a test failure.  There's no test failure relating to what you say.